site/public/css/polls.css

```diff
@@ -64,6 +69,11 @@ input[type="radio"][disabled=""]:checked::before {
     background: var(--standard-medium-dark-gray);
 }
 
+input[type="checkbox"] {
+    margin: 0;
+    transform: scale(0.8);
+}
```

Comment:
It would probably be better to target a specific class given to the relevant inputs instead of targeting all checkboxes.  If we decide to add another checkbox to the UI somewhere, this will cause confusion.

Reply:
Done. Now it only targets checkboxes whose name = "answer[ ]", which was originally defined in the file.